### ISSUE TITLE
Co-locating imported CSS files with the PostCSS plugin

### ISSUE THREAD

**Issue by cgwrench** [lumeland/lume#485]
I'm getting started with Lume and trying to locate `@import`-ed CSS files in the same directory as the files that import them. I'm using the PostCSS plugin.

All the examples and documentation I've found so far place the imported files in a different directory, usually in a sub-directory of `_includes`. Is there a way to have CSS files and the files they import in the same directory?

For example, the [Lume base blog example](https://github.com/lumeland/base-blog) has a main CSS file, `styles.css`, in the root of the repository, which imports other CSS files from an `_includes/css` directory. These files are structured as follows:

```
├── _includes
│   └── css
│       ├── navbar.css
│       ├── post-list.css
│       └── …
└── styles.css
```

`styles.css` contains `@import` at-rules like:

```css
…
@import "css/reset.css";
@import "css/navbar.css";
…
```

I'd prefer to have files structured as follows:

```
└── css
    ├── navbar.css
    ├── post-list.css
    ├── …
    └── styles.css
```

And my `styles.css` would contain `@import` at-rules like:

```css
…
@import "reset.css";
@import "navbar.css";
…
```

In other words, the imports are resolved relative to the file containing the imports, not relative to the `_includes` directory.

I've managed to get this working with the following configuration:

```js
site
  .use(postcss({
    includes: "css/" // trailing '/' matters here
  }))
```

Note that the trailing `/` matters here - without it, no CSS files are created in the `_site` directory. This feels like a bit of a hack - is this the way to achieve the file structure I'm looking for, or is there a better way?

**Comment by oscarotero** [comment]
To import files relative to the current file, use relative paths (starting with dot):

```css
@import "./reset.css";
@import "./navbar.css";
```

Note that Lume will load and process all css files found in the src directory, so you will have not only the `styles.css` files but also the `reset.css` and `navbar.css` files in the dest folder. That's why they are placed in the `_includes` folder. because it's ignored due it starts with `_`. You can [ignore the other files](https://lume.land/docs/configuration/ignore-files/) with this function:

```js
// Ignore all css files but "styles.css"
site.ignore((path) => path.endsWith(".css") && path !== "/styles.css");
```
